content/guides/hot-module-replacement.md

```diff
@@ -195,33 +209,82 @@ Now let's update the configuration file to make use of the loader.
       new webpack.HotModuleReplacementPlugin()
     ],
     output: {
-      filename: '[name].bundle.js',
+      filename: 'bundle.js',
       path: path.resolve(__dirname, 'dist')
     }
   };
 ```
 
-hot loading stylesheets is a breeze...
+Hot loading stylesheets is as easy as importing them into a module:
 
-__index.js__
-
-``` js
-import printMe from './print.js';
-import './styles.css';
+__project__
 
-// ...
+``` diff
+  webpack-demo
+  | - package.json
+  | - webpack.config.js
+  | - /dist
+    | - bundle.js
+  | - /src
+    | - index.js
+    | - print.js
++   | - styles.css
 ```
 
 __styles.css__
 
 ``` css
 body {
-  background: blue;
+  background: red;
```

Comment:
This should be `background: blue;` to make it line up with the diff further down ;)